Makefile

```diff
@@ -1,4 +1,4 @@
--include .env .env.local
+#-include .env .env.local
```

Comment:
? why

Reply:
Windows Docker setup has some issue reading env and commented, to proceed further, I will enable it before keeping PR ready to review

Reply:
oh ????

Reply:
yes now resolved